bundles/org.openhab.binding.bluetooth.bluez/src/main/resources/ESH-INF/thing/bluez.xml

```diff
@@ -15,24 +14,6 @@
 				<label>Address</label>
 				<description>The Bluetooth address of the adapter in format XX:XX:XX:XX:XX:XX</description>
 			</parameter>
-			<parameter name="backgroundDiscovery" type="boolean">
-				<label>Background Discovery</label>
-				<description>Whether this adapter performs background discovery of Bluetooth devices</description>
-				<advanced>true</advanced>
-				<default>false</default>
-			</parameter>
-			<parameter name="inactiveDeviceCleanupInterval" type="integer" min="1" unit="s">
-				<label>Device Cleanup Interval</label>
-				<description>How often device cleanup is performed</description>
-				<advanced>true</advanced>
-				<default>60</default>
-			</parameter>
-			<parameter name="inactiveDeviceCleanupThreshold" type="integer" min="1" unit="s">
-				<label>Device Cleanup Threshold</label>
-				<description>Timespan a device can remain radio silent before it is eligible for cleanup</description>
-				<advanced>true</advanced>
-				<default>300</default>
-			</parameter>
```

Comment:
This should be unchanged, these config features are present in all adapter implementations.